applications/stepswitch/src/stepswitch_resources.erl

```diff
@@ -584,7 +602,8 @@ fetch_local_resources(AccountId) ->
         {'ok', JObjs} ->
             CacheProps = [{'origin', fetch_local_cache_origin(JObjs, AccountDb, [])}],
             Resources = resources_from_jobjs([wh_json:get_value(<<"doc">>, JObj) || JObj <- JObjs]),
-            LocalResources = [Resource#resrc{global='false'} || Resource <- Resources],
+            Proxies = fetch_account_dedicated_proxies(AccountId),
```

Comment:
Perhaps we should also pass the `global` boolean in? So global resources will call `resources_from_jobjs/1` which will call `resources_from_jobj(Resources, 'true', [])`. Meanwhile local resources will call `resources_from_jobjs(Resources, 'false', Proxies)`.

I'm just looking to get rid of unnecessary iteration; if this isn't too cumbersome to do: great!